### PR TITLE
For DS-2876 adding country/plan interval/plan interval count

### PR DESCRIPTION
**Pull request by YeonjooSmith**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/mozilla_vpn_derived/channel_group_proportions_live/view.sql [hunk 1]
```diff
@@ -5,6 +5,7 @@ WITH stage_1 AS (
   SELECT
     event_date AS subscription_start_date,
     country_name,
+    country,
```
Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/mozilla_vpn_derived/channel_group_proportions_live/view.sql:8]: Select column order needs to match what's in `schema.yaml`.  I think the order in `schema.yaml` makes more sense, so I suggest changing this.
```suggestion
    country,
    country_name,
```

File: sql/moz-fx-data-shared-prod/mozilla_vpn_derived/channel_group_proportions_live/view.sql [hunk 2]
```diff
@@ -25,6 +28,7 @@ WITH stage_1 AS (
   GROUP BY
     subscription_start_date,
     country_name,
+    country,
```
Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/mozilla_vpn_derived/channel_group_proportions_live/view.sql:31]: Group by column order doesn't really matter, but for consistency with the above change...
```suggestion
    country,
    country_name,
```

**Review by sean-rose — APPROVED** [review]
r+wc

**Comment by sean-rose** [comment]
Merging this is getting blocked on a CI issue.  I'll investigate tomorrow.

**Comment by sean-rose** [comment]
The `mozilla_vpn_derived.channel_group_proportions_v1` table has been rebuilt with this change.

**Comment by sean-rose** [comment]
The Looker view has now also been updated to include the new columns.

**Comment by YeonjooSmith** [comment]
Thank you for the update!!

**Pull request merged**